dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/PavedRoadFramework.kt

```diff
@@ -24,7 +24,7 @@ import java.io.File
  * A PavedRoadFramework is the simplest way to integrate a new framework into Dataland or
  * update an existing one! It provides a template for implementing frameworks.
  */
-@Suppress("TooManyFunctions")
+@Suppress("TooManyFunctions", "LongParameterList")
```

Comment:
Does that come from manual maintentance? If so, shouldn't we rather change the configuration instead of supressing this everywhere?

Reply:
Ich glaube das ist in main in sonar auch schon angemeckert.
I-jemand hat das mal in sonar manuell auf der sonarcloud website auf "false positive" oder so gesetzt, und dann ist das in main gelandet.

Vermutlich wollte Marcel das hier einfach nicht wieder entstehen lassen.

Ich glaube er hatte das mit Siegfried diskutiert.

Reply:
Verstehe. Ich dachte 6 Parameter wäre gerade so in ordnung.